frontend/src/lib/k8s/ingress.ts

```diff
@@ -92,31 +92,35 @@ class Ingress extends makeKubeObject<KubeIngress>('ingress') {
     const rules: IngressRule[] = [];
 
     this.spec!.rules?.forEach(({ http, host }) => {
-      const paths = http.paths.map(({ backend, path }) => {
-        if (!!(backend as LegacyIngressBackend).serviceName) {
-          return {
-            path,
-            backend: {
-              service: {
-                name: (backend as LegacyIngressBackend).serviceName,
-                port: {
-                  number: parseInt((backend as LegacyIngressBackend).servicePort, 10),
+      if (http) {
```

Comment:
I think it should still display the rule if it doesn't have http paths. This would currently just not display the whole rule.

Reply:
Set the rule to display with empty paths list, as well as the `console.debug` line to keep track. Also added the case without HTTP paths to a story, I believe the other stories should cover the other case